h2o-bindings/bin/gen_R.py

```diff
@@ -65,6 +65,8 @@ def gen_module(schema, algo, module):
         if param["name"] == "seed":
             if algo != "stackedensemble":
                 yield "#' @param seed Seed for random numbers (affects certain parts of the algo that are stochastic and those might or might not be enabled by default)"
+                yield "#'        The type is long, however, R cannot handle long number correctly, so the type can also be a string."
```

Comment:
I would leave the function documentation as is. The matter not very clear to a regular user and also won't affect the regular user. Instead of having the topic explained in each algo, I would prefer to put it in a small section in the documetation.